core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -82,11 +82,19 @@ public O execute(ClientRequestContext ctx, I req) throws Exception {
      */
     protected final O executeDelegate(ClientRequestContext ctx, I req) throws Exception {
         final ClientRequestContext derivedContext = ctx.newDerivedContext();
+        ctx.logBuilder().addChild(derivedContext.log());
         try (SafeCloseable ignore = RequestContext.push(derivedContext, false)) {
             return delegate().execute(derivedContext, req);
         }
     }
 
+    /**
+     * This should be called when retrying is finished.
+     */
+    protected static void onRetryingComplete(ClientRequestContext ctx) {
```

Comment:
Is there any reason making this method `static` unlike the other methods such as `setResponseTimeout`, `getNextDelay`, etc?